packages/payment-widget/src/lib/utils/chains.ts

```diff
@@ -0,0 +1,93 @@
+export const chains = [
+  {
+    chainId: 1,
+    name: "Ethereum",
+    currency: "ETH",
+    explorerUrl: "https://etherscan.io",
+    rpcUrl: "https://cloudflare-eth.com",
+  },
+  {
+    chainId: 11155111,
+    name: "Sepolia",
+    currency: "ETH",
+    explorerUrl: "https://sepolia.etherscan.io",
+    rpcUrl: "https://rpc.sepolia.org",
+  },
+  {
+    chainId: 137,
+    name: "Polygon",
+    currency: "MATIC",
+    explorerUrl: "https://polygonscan.com",
+    rpcUrl: "https://polygon.llamarpc.com",
+  },
+  {
+    chainId: 56,
+    name: "Binance Smart Chain",
+    currency: "BNB",
+    explorerUrl: "https://bscscan.com",
+    rpcUrl: "https://binance.llamarpc.com",
+  },
+  {
+    chainId: 100,
+    name: "Gnosis Chain (xDai)",
+    currency: "XDAI",
+    explorerUrl: "https://gnosis.blockscout.com/",
+    rpcUrl: "https://gnosis-rpc.publicnode.com",
+  },
+  {
+    chainId: 43114,
+    name: "Avalanche",
+    currency: "AVAX",
+    explorerUrl: "https://snowtrace.io",
+    rpcUrl: "https://1rpc.io/avax/c",
+  },
+  {
+    chainId: 10,
+    name: "Optimism",
+    currency: "ETH",
+    explorerUrl: "https://optimistic.etherscan.io",
+    rpcUrl: "https://mainnet.optimism.io",
+  },
+  {
+    chainId: 42161,
+    name: "Arbitrum One",
+    currency: "ETH",
+    explorerUrl: "https://arbiscan.io",
+    rpcUrl: "https://arb1.arbitrum.io/rpc",
+  },
+  {
+    chainId: 250,
+    name: "Fantom",
+    currency: "FTM",
+    explorerUrl: "https://ftmscan.com",
+    rpcUrl: "https://rpc.ftm.tools",
+  },
+  {
+    chainId: 42220,
+    name: "Celo",
+    currency: "CELO",
+    explorerUrl: "https://explorer.celo.org/mainnet/",
+    rpcUrl: "wss://celo.drpc.org",
+  },
+  {
+    chainId: 1284,
+    name: "Moonbeam",
+    currency: "GLMR",
+    explorerUrl: "https://moonbeam.moonscan.io",
+    rpcUrl: "https://rpc.api.moonbeam.network",
+  },
+  {
+    chainId: 324,
+    name: "ZkSync Era",
+    currency: "ETH",
+    explorerUrl: "https://explorer.zksync.io",
+    rpcUrl: "https://mainnet.era.zksync.io",
+  },
+  {
+    chainId: 8453,
+    name: "Base",
+    currency: "ETH",
+    explorerUrl: "https://basescan.org",
+    rpcUrl: "https://mainnet.base.org",
+  },
+];
```

Comment:
If we have a lot of users, we may have a rate limit on these public RPC endpoints, but let's see and leave it for V2